docs/diagrams/PlantUML File/FindClientSequenceDiagram.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
+
+hide footbox
+
+Participant ":CommandFindClient" as CommandFindClient #F4F4B6
+Participant "clientList : ClientList" as ClientList #F4F4B6
+Participant "client : Client" as Client #F4F4B6
+Participant "ui: Ui" as Ui #F4F4B6
+
+->CommandFindClient:execute(Ui:ui, Storage: storage, PropertyList: propertyList, \nClientList: clientList, PairingList: pairingList
```

Comment:
I think the format for arguments in method calls should be `methodName(<variable name>: <variable type>)`
```
execute(ui: Ui, storage: Storage, propertyList: PropertyList ...
```